### DISCUSSION TITLE
Browser Port

### DISCUSSION THREAD

**Discussion by xiao2718** [meikpiep/gauguin#142] [Ideas]
I absolutely love the game, but have you yet had time to think about a desktop/Browser port?

**Comment by meikpiep** [comment]
Hi there, not really. I have plenty to do regarding the original Android version. :-)

I've thought that may be the existing Kotlin code can be used quiet easily via Kotlin Muliplatform (KMP) to create a desktop version. But this is only a thought.

To be honest, I don't know if I would do a desktop or browser version if the to do list was shorter.
